devops/actions/e2e-tests-win/action.yml

```diff
@@ -30,7 +30,7 @@ runs:
     run: |
       mkdir build-e2e
       set PATH=%GITHUB_WORKSPACE%\install\bin;%PATH%
-      cmake -GNinja -B build-e2e -S.\llvm\sycl\test-e2e -DSYCL_TEST_E2E_TARGETS="${{ inputs.targets }}" -DCMAKE_CXX_COMPILER="clang++" -DLLVM_LIT="..\llvm\llvm\utils\lit\lit.py" ${{ inputs.cmake_args }}
+      cmake -GNinja -B build-e2e -S.\llvm\sycl\test-e2e -DSYCL_TEST_E2E_TARGETS="${{ inputs.targets }}" -DCMAKE_CXX_COMPILER="clang++" -DLEVEL_ZERO_LIBS_DIR="D:\github\level-zero_win-sdk\lib" -DLLVM_LIT="..\llvm\llvm\utils\lit\lit.py" ${{ inputs.cmake_args }}
```

Comment:
I guess I'm not familiar with this macro.  Doesn't this tell the test suite where to find the L0 Dev kit?  Why would the L0 devkit be at this location?

Reply:
Yes, since we do not want to install the L0 loader, we require end-users install it themselves (same way we require L0 drivers). The path I used is where our Windows CI had been configured to have the L0 loader in.

Reply:
Actually I am closing this PR in favor of https://github.com/intel/llvm/pull/9166/ carrying the same change.